src/main/java/zaasclient/ConfigProperties.java

```diff
@@ -0,0 +1,113 @@
+/*
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ */
+package zaasclient;
+
+public class ConfigProperties {
+
+    private String apimlHost;
+    private String apimlPort;
+    private String apimlBaseUrl;
+    private String keyStoreType;
+    private String keyStorePath;
+    private String keyStorePassword;
+    private String trustStoreType;
+    private String trustStorePath;
+    private String trustStorePassword;
+
```

Comment:
This constructor can be removed. Java will create an empty constructor, if no other constructor is specified (such as this case)